server/service/hosts.go

```diff
@@ -718,11 +718,17 @@ func (svc *Service) MacadminsData(ctx context.Context, id uint) (*fleet.Macadmin
 		return nil, err
 	}
 
+	var munkiInfo *fleet.HostMunkiInfo
 	version, err := svc.ds.GetMunkiVersion(ctx, id)
```

Comment:
Nit: I'm used to using a switch in these cases:
```
switch version, err := svc.ds.GetMunkiVersion(ctx, id); {
case err == nil:
  munkiInfo = &fleet.HostMunkiInfo{Version: version}
case fleet.IsNotFound(err):
  // OK
default:
  return nil, err
}
```

Reply:
Agreed!